src/upkeeps/UpKeepManager.sol

```diff
@@ -0,0 +1,497 @@
+    /// @notice only callable via governance
+    /// @param _memberAddress contract address to be cancel upkeep
+    function cancelMemberUpKeep(address _memberAddress) external onlyGovernance {
+        if (!_members.contains(_memberAddress)) revert NotMemberIncluded(_memberAddress);
+
+        // NOTE: only member which upkeep is being cancelled can be removed
+        uint256 upKeepId = membersInfo[_memberAddress].upKeepId;
+        CL_REGISTRY.cancelUpkeep(upKeepId);
+    }
+
+    /// @dev Withdraws LINK funds and remove member from manager
+    /// @notice only callable via governance
+    /// @param _memberAddress contract address to be remove from manager
+    function withdrawLinkFundsAndRemoveMember(address _memberAddress) external onlyGovernance {
+        if (!_members.contains(_memberAddress)) revert NotMemberIncluded(_memberAddress);
+
+        uint256 upKeepId = membersInfo[_memberAddress].upKeepId;
+        _members.remove(_memberAddress);
+        delete membersInfo[_memberAddress];
+
+        // NOTE: only member which upkeep is being cancelled can be removed
+        (,,, uint96 linkRefund,,, uint64 maxValidBlocknumber,) = CL_REGISTRY.getUpkeep(upKeepId);
+        // https://etherscan.io/address/0x02777053d6764996e594c3e88af1d58d5363a2e6#code#F1#L738
+        if (maxValidBlocknumber == UINT64_MAX) revert UpKeepNotCancelled(upKeepId);
+
+        CL_REGISTRY.withdrawFunds(upKeepId, address(this));
+
+        emit RemoveMember(_memberAddress, upKeepId, linkRefund, block.timestamp);
+    }
+
+    /// @dev  Sweep the full LINK balance to techops
+    function sweepLinkFunds() external onlyGovernance {
+        uint256 linkBal = LINK.balanceOf(address(this));
+        LINK.transfer(TECHOPS, linkBal);
+        emit SweepLinkToTechops(linkBal, block.timestamp);
+    }
+
+    /// @dev  Sweep the full ETH balance to recipient
+    /// @param _recipient Address receiving eth funds
+    function sweepEthFunds(address payable _recipient) external onlyGovernance {
+        uint256 ethBal = address(this).balance;
+        _recipient.transfer(ethBal);
+        emit SweepEth(_recipient, ethBal, block.timestamp);
+    }
+
+    ////////////////////////////////////////////////////////////////////////////
+    // PUBLIC: Governance - Pausing
+    ////////////////////////////////////////////////////////////////////////////
+
+    /// @dev Pauses the contract, which prevents executing performUpkeep.
+    function pause() external onlyGovernance {
+        _pause();
+    }
+
+    /// @dev Unpauses the contract.
+    function unpause() external onlyGovernance {
+        _unpause();
+    }
+
+    ////////////////////////////////////////////////////////////////////////////
+    // PUBLIC: Governance - Config
+    ////////////////////////////////////////////////////////////////////////////
+
+    /// @notice Updates the value of `roundsTopUp`, which is used for decided how much rounds will
+    ///         will be covered at least while topping-up
```

Comment:
will will

Reply:
removed in b79599ae74e183e1dda6bb469de6f90330dbcfa5